src/bentoml_cli/deployment.py

```diff
@@ -211,6 +211,11 @@ def deployment_cli():
         type=click.Path(exists=True),
         help="Path to the project",
     )
+    @click.argument(
+        "name",
+        type=click.STRING,
+        required=True,
+    )
```

Comment:
I didn't get why this is added but the args are removed from the function

Reply:
By adding args we can use cli like this:
```
bentoml deployment update my-deploy --scaling-min=1
```

Reply:
Oh, I am looking at the wrong line.